contracts/Policy.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+/**
+ * @dev Object defining how a table can be accessed.
+ */
+struct Policy {
```

Comment:
i wonder, instead of pulling this out of the `ITablelandController`, if we should also version the Policy? if we eventually have to version `Policy`, it might be cumbersome to have to map non-matching versions of controller and policy.

Reply:
As noted in the issue #300, the compiler doesn't like the implicit conversion in between even identical structs, so different versions of structs will probably be separate from controller version interfaces.

Reply:
did you see this comment? https://github.com/tablelandnetwork/evm-tableland/issues/156#issuecomment-1435150250. ie, you could do an explicit conversion.

Reply:
I've updated it. I mapped the ITablelandControllerV1 policy to the ITablelandController.Policy struct; I like having all of it in the latest interface, reduces having to hop around to figure out the current, correct interface. Though it does seem a little weird to have a V1 interface pointing to a V2 struct (though technically it hasn't changed).